src/app/course/_test/mock.ts

```diff
@@ -316,3 +317,94 @@ export const MOCK_UQJ2: UQJ = {
     variables_errors: [],
     is_checkbox: false,
 }
+
+export const MOCK_USER_STATS1: Stats = {
+    challenge_stats: {
+        challenges_completed: 2
+    },
+    goal_stats: {
+        goals_completed: 2
+    },
+    question_stats: {
+        mcq: {
+            questions_attempt: 0,
+            questions_solved: 2,
+            avgSuccess: 0
```

Comment:
Some of this data doesn't make sense.
You don't need them here but it might be used later by someone else and then they have to fix your stuff.

questions_attempt cannot be less than questions_solved.
and avgSuccess should be equal to questions_solved / questions_attempt